tests/test_app.py

```diff
@@ -257,10 +257,11 @@ def test_initialize_with_admin_dashboard(test_model):
     Args:
         test_model: The default model.
     """
-    app = App(admin_dash=AdminDash(models=[test_model]))
-    assert app.admin_dash is not None
-    assert len(app.admin_dash.models) > 0
-    assert app.admin_dash.models[0] == test_model
+    print(test_model)
+    App(admin_dash=AdminDash(models=[test_model]))
+    # assert app.admin_dash is not None
+    # assert len(app.admin_dash.models) > 0
+    # assert app.admin_dash.models[0] == test_model
```

Comment:
can we uncomment these now?
```suggestion
    assert app.admin_dash is not None
    assert len(app.admin_dash.models) > 0
    assert app.admin_dash.models[0] == test_model
```